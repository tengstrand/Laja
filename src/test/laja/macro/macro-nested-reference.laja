
#macro one(argument)
1{argument}##
#end

#macro two(argument)
2{argument}##
#end

#write "target/test-output/macro-nested-reference.out"
  #set (aaa = "AAA")
{one(two(aaa))}
{one(two(aaa.class.name))}
{one(two("3{aaa}"))}
#end
