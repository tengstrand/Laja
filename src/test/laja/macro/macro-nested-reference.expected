12AAA
12java.lang.String
123AAA
